lib/srv/sess.go

```diff
@@ -1681,16 +1627,8 @@ func (p *party) String() string {
 }
 
 func (p *party) Close() (err error) {
-	p.closeOnce.Do(func() {
-		p.log.Infof("Closing party %v", p.id)
-		if err = p.s.registry.leaveSession(p); err != nil {
-			p.ctx.Error(err)
-		}
-		close(p.closeC)
-		close(p.termSizeC)
-		err = p.ch.Close()
-	})
-	return err
+	p.log.Infof("Closing party %v", p.id)
```

Comment:
Are we sure that this is never called twice?

Reply:
Yes this is the case now, as it is only called in `removeParty` which is performed behind the session mutex on `s.parties`.

Reply:
👍